documentation/docs/contribute.md

```diff
@@ -9,35 +9,36 @@ keywords:
 - Identity Initiative
 - Documentation
 - Discord
+- reference
 ---
 
-**Thanks for thinking about contributing to the project! We have the following ways that you can contribute.**
+**Thanks for thinking about contributing to the project! You can contribute using the following methods.**
 
 ## Join the Identity Initiative
 
-The [Identity Initiative](https://github.com/iota-community/X-Team_IOTA_Identity) is a collaborative effort to help improve the developer experience.
+The [Identity Initiative](https://github.com/iota-community/X-Team_IOTA_Identity) is a collaborative effort to help improve the developer experience that includes:
 
-- Quality assurance and review
-- Documentation
-- Code samples
+- Quality assurance and review.
+- Documentation.
+- Code samples.
 
-If you'd like to get involved, join the #experience channel on [Discord](https://discord.iota.org).
+If you would like to get involved, join the #experience channel on [Discord](https://discord.iota.org).
```

Comment:
```suggestion
If you would like to get involved, join the [#x-team-identity](https://discord.com/channels/397872799483428865/773274309861834782) channel on [Discord](https://discord.iota.org).
```

Reply:
I don't think it will take you directly to the channel unless you already have joined the server.

Reply:
👌